pootle/apps/pootle_store/views.py

```diff
@@ -460,9 +460,10 @@ def get_context_data(self, *args, **kwargs):
             'has_admin_access': check_user_permission(self.request.user,
                                                       'administrate',
                                                       self.directory),
-            'altsrcs': self.get_alt_srcs(),
+            'altsrcs': dict([(x.id, x.data) for x in self.get_alt_srcs()]),
```

Comment:
Isn't a dict comprehension useful and more readable here? `{x.id: x.data for x in self.get_alt_srcs()}`